### ISSUE TITLE
Build failed, app/build/intermediates/transforms/Instabug is defined multiple times

### ISSUE THREAD

**Issue by Gosunet** [Instabug/Instabug-Android#444]
Hello, we are using instabug in two apps (a legacy one and a new one) and sometimes when we build our app we have this kind of error during the `dexBuilder` gradle task :

```
Type androidx.compose.compiler.plugins.kotlin.ModuleMetricsImpl$saveReportsTo$3 is defined multiple times: .../app/build/intermediates/transforms/Instabug/preprod/debug/249/androidx/compose/compiler/plugins/kotlin/ModuleMetricsImpl$saveReportsTo$3.class, .../app/build/intermediates/transforms/Instabug/preprod/debug/338/androidx/compose/compiler/plugins/kotlin/ModuleMetricsImpl$saveReportsTo$3.class
```

We have to delete the folder `/app/build/intermediates/transforms/Instabug` to make it work again. It usually happen when switching branche or variant.

### Steps to Reproduce the Problem

I don't know.
But it usually happended when switching branch then building the app after having build the app in the first branch.

### Expected Behavior

It should build without error

### Actual Behavior

We got an error during build

### Instabug integration code

```
 Instabug.Builder(application, apiKey, InstabugInvocationEvent.SCREENSHOT).build()
            rejectConsentByDefault()
            setupApplicationPerformanceMonitoring()
            setupBugReporting()
```
    

### SDK Version

instabug = "11.7.0"

### Android Version

compileSdkVersion = 33
minSdkVersion = 21
targetSdkVersion = 33

### [Optional] Project That Reproduces the Issue

**Comment by maxharter** [comment]
Hi Instabug team,
@IslamMohamady 
Do you have any news about this issue please?
Thanks,

Maxime

**Comment by IslamMohamady** [comment]
Hi @maxharter 
This should be fixed in latest release 11.10.0
I will close the ticket for now. And feel free to reopen it if you have any inquiry
